core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -297,7 +297,7 @@ protected void startServingRPCServer() {
       if (mTServerSocket != null) {
         mTServerSocket.close();
       }
-      mTServerSocket = new TServerSocket(mRpcAddress,
+      mTServerSocket = new TServerSocket(NetworkAddressUtils.getBindAddress(ServiceType.MASTER_RPC),
```

Comment:
@aaudiber turns out we need the variable otherwise it does not work for unit tests (we will resolve a new port)